sips/sip-111.md

```diff
@@ -0,0 +1,76 @@
+
+From [SIP-93](./sip-93.md) the Spartan Council was established, during this initial phase, they were proposed to vote only on low-level governance SCCP changes. At the time of this SIP proposal, the Spartan Council has successfully voted on more than 20 SCCPs and has shown positive engagement with the community and other stakeholders in debating and responding to issues that have arisen in the protocol.
+
+As a result of this positive outcome, it is proposed that the Spartan Council should start voting on SIPs as the current SIP process is still carried out in a weaker version of governance within the #governance-poll discord channel which still possesses many of the issues explored in SIP-93.
+
+## Specification
+
+<!--The specification should describe the syntax and semantics of any new feature, there are five sections
+1. Overview
+2. Rationale
+3. Technical Specification
+4. Test Cases
+5. Configurable Values
+-->
+
+### Overview
+
+<!--This is a high level overview of *how* the SIP will solve the problem. The overview should clearly describe how the new feature will be implemented.-->
+
+If this SIP is implemented, all proposed SIPs will be created within the https://gov.synthetix.io IPFS snapshot space and governance discussions within the #council-chambers will include these SIPs.
+
+### Rationale
+
+<!--This is where you explain the reasoning behind how you propose to solve the problem. Why did you propose to implement the change in this way, what were the considerations and trade-offs. The rationale fleshes out what motivated the design and why particular design decisions were made. It should describe alternate designs that were considered and related work. The rationale may also provide evidence of consensus within the community, and should discuss important objections or concerns raised during discussion.-->
+
+There have been a number of discussions around when to transition SIPs to the Spartan Council, this SIP has been proposed as a standalone vote as opposed to SIP-93/SIP-104 which tied this decision into other changes to the protocol. Given the importance of this change, it is ideal to isolate it from any other decisions and address it as a separate SIP. The changes proposed to the governance process in SIP-104 are still approved but would no longer be a dependency for this change.
+
+### Technical Specification
+
+<!--The technical specification should outline the public API of the changes proposed. That is, changes to any of the interfaces Synthetix currently exposes or the creations of new ones.-->
+
+All SIPs that are created on the SIP GitHub repo will need to be accompanied by a snapshot proposal on https://gov.synthetix.io/
+
```

Comment:
In addition, prior to a proposal being uploaded to gov.synthetix.io a SIP presentation must be held and attended by the SIP author, core contributors assigned to implement the SIP and at least 50% of the council. This presentation will use Discord voice chat to allow anon members to not doxx themselves. It will provide a forum for any considerations and issues arising from the SIP to be interrogated and raised prior to a formal vote by the Council.